deployments/main.tf

```diff
@@ -0,0 +1,95 @@
+variable "aiven_api_token" {
+  type = string
+}
+
+variable "aiven_project_name" {
+  type = string
+}
+
+variable "aiven_service_name" {
+  type = string
+}
+
+variable "aiven_service_plan" {
+  type = string
+}
+
+variable "aiven_kafka_cloud_name" {
+  type = string
+}
+
+variable "aiven_kafka_topic_name" {
+  type = string
+}
+
+variable "aiven_kafka_username" {
+  type = string
+}
```

Comment:
Moving all the input variables in a dedicated `variables.tf` file could make the configuration easier to read

Reply:
👍  Yup totally, good point. (In fact I moved them in a new PR I'm working on for adding the tf files for the event-gateway demo app, but better to do it right now here).

It's done. Also I moved all those files into a subdir `kafka-test` as more modules will come in new pr's soon.

Reply:
I took the opportunity to also convert to vars 3 values:

`aiven_kafka_topic_partitions`, `aiven_kafka_topic_replication`, and `aiven_kafka_topic_retention_ms`